codebase/billing-service-secondary-sync/src/main/scala/com/virtuslab/billingsync/Config.scala

```diff
@@ -0,0 +1,8 @@
+package com.virtuslab.billingsync
```

Comment:
isn't it simpler to have everything in single config? (just add these two new props there?)

Reply:
done